app/controllers/concerns/project_concern.rb

```diff
@@ -0,0 +1,13 @@
+module ProjectConcern
+  extend ActiveSupport::Concern
+
+  private
+
+  def change_date(project)
```

Comment:
I like inheritance. I will see what @hooverlunch says and we will go with that. Thanks, @linesarefuzzy.